src/main/java/com/exercise/sfe/repository/TagRepository.java

```diff
@@ -0,0 +1,62 @@
+package com.exercise.sfe.repository;
+
+import com.exercise.sfe.constant.Fields;
+import com.exercise.sfe.entity.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+
+@Repository
+@RequiredArgsConstructor
+public class TagRepository implements DefaultRepository<Tag> {
+
+    private JdbcTemplate jdbcTemplate;
```

Comment:
Please read java code conventions, section class. What is recommended way of class structure